src/IconToggle/index.js

```diff
@@ -2,13 +2,36 @@ import React, { Component } from 'react'
 import { View, StyleSheet } from 'react-native'
 import { IconToggle } from '@protonapp/react-native-material-ui'
 
+
+
 export default class WrappedIconToggle extends Component {
   handlePress = async () => {
     let {
       clickActions,
       input: { value, onChange },
     } = this.props
-    await onChange(!value)
+    //await onChange(!value)
+
+    let {
+      activeActions,
+      input: { value, onChange },
+    } = this.props
+    await onChange(value == true)
+    if (activeActions) {
+      await activeActions()
+    }
+    if (clickActions) {
+      await clickActions()
+    }
+
+    let {
+      inactiveActions,
+      input: { value, onChange },
+    } = this.props
+    await onChange(value == false )
+    if (inactiveActions) {
```

Comment:
```suggestion
    if (inactiveActions && value) {
```
Similar to my suggestion on line 20